Sources/App/Core/SearchFilter.swift

```diff
@@ -338,32 +320,14 @@ struct LastCommitSearchFilter: SearchFilter {
 /// last_activity:<2021-10-01 - Last maintenance activity on any day older than November 1st 2021
 /// ```
 struct LastActivitySearchFilter: SearchFilter {
-    static var parseDateFormatter: DateFormatter = {
-        let formatter = DateFormatter()
-        formatter.dateFormat = "yyyy-MM-dd"
-        formatter.locale = Locale(identifier: "en_US_POSIX")
-        formatter.timeZone = TimeZone(secondsFromGMT: 0)
-
-        return formatter
-    }()
-
-    static var viewDateFormatter: DateFormatter = {
-        let formatter = DateFormatter()
-        formatter.dateFormat = "d MMM yyyy"
```

Comment:
Personally I find it a bit odd that the display format is different from the supported search syntax. I guess it does tell the user that the parsing was ok but it also suggests that they should use the display format (which wouldn't work, or would it?).

I think I'd have stuck with `yyyy-MM-dd` in both cases. It's the unambiguous "programmery" date format we're all comfortable with.

Reply:
I'm not sure I agree here. The reason we have the YYYYMMDD format in the filter is for ease of parsing, but I don't think it's necessarily a good thing that we use that elsewhere. In fact, I think given that the formatted output here is using a month name is a great validation of what date is being parsed out.

I'd vote to keep it like this.

Reply:
I can definitely see your perspective. I changed the display format not really as a way of proving it worked but more of a validation to the user that they got the correct format. I'll be the first to admit I often get the MM and DD mixed up and it could return confusing results otherwise.

One potential option is to have some form of tooltip perhaps? So the display format is the raw input (YYYY-MM-DD) but if you hover over it then you see the formatter version as a validation?

Reply:
It's just that it leads to

<img width="449" alt="CleanShot 2021-12-01 at 14 52 22@2x" src="https://user-images.githubusercontent.com/65520/144246491-34147efd-71e1-4724-bd58-147a09cf9995.png">

or, slightly weirder

<img width="376" alt="CleanShot 2021-12-01 at 14 53 06@2x" src="https://user-images.githubusercontent.com/65520/144246578-266e9bae-7cd7-48a4-9319-6fbb1e1e9449.png">

(where I input `test last_commit:>"1 Nov 2021"` and it gets truncated to `test last_commit:>`)

🤷‍♂️

Reply:
> (where I input test last_commit:>"1 Nov 2021" and it gets truncated to test last_commit:>)

That's a great bug, but unrelated to this change I think.

I'd still vote to keep the formatting like it is here, but both of these issues are a little unrelated to this PR as the date filters already work like this.

I'm going to merge this PR. If you feel strongly that we should change it then let's log it as a bug and discuss it further in a separate issue.

Reply:
> That's a great bug, but unrelated to this change I think.

Yes, it's unrelated. James and I discussed that briefly in the original PR where we noticed this behaviour.

I'm not too fussed, just wanted to point out the issue and had ticked the approve already :)